src/main/java/org/jabref/logic/importer/fileformat/MarcXmlParser.java

```diff
@@ -128,16 +133,17 @@ private BibEntry parseEntry(Element element) {
 
         /*
          * ToDo:
-         *  pages
-         *  volume and number correct
-         *  series and journals stored in different tags
-         *  thesis
-         *  proceedings
+         *  pages //Done
+         *  volume and number correct //Done
+         *  series and journals stored in different tags //Done?
```

Comment:
Please, don't track TODOs in code. - You can add a markdown checklist to the PR description and then add to the chck boxes.

Example:

```markdown
- [ ] open todo
- [x] done
```